src/app/home/home.routes.ts

```diff
@@ -4,19 +4,19 @@ import { DefaultComponent } from './default/default.component';
 import { PlayEpisode } from './play-episode/play-episode.component';
 import { BangumiList } from './bangumi-list/bangumi-list.component';
 import { BangumiDetail } from './bangumi-detail/bangumi-detail.components';
-import { Authentication } from '../user-service/authentication.service';
 import { UserCenter } from './user-center/user-center.component';
 import { WebHookComponent } from './web-hook/web-hook.component';
 import { PreviewVideoComponent } from './preview-video/preview-video.component';
 import { FavoriteListComponent } from './favorite-list/favorite-list.component';
 import { BangumiAccountBindingComponent } from './bangumi-account-binding/bangumi-account-binding.component';
 import { MyHistoryComponent } from './my-history/my-history.component';
+
 export const homeRoutes: Routes = [
     {
         path: '',
         component: Home,
         data: {level: 0},
-        canActivate: [Authentication],
+        // canActivate: [Authentication],
```

Comment:
Why is this line removed

Reply:
Currently, I haven't come up with a replacement for route guard. I should add a TODO in comment here.

Reply:
Added TODO here